src/components/swapv2/SingleTokenInfo.tsx

```diff
@@ -74,32 +73,28 @@ const AboutText = styled.h2`
 `
 
 const LINE_HEIGHT = 24
+const HEIGHT = 280
 const DescText = styled(InfoRowLabel)<{ showLimitLine: boolean }>`
+  line-height: ${LINE_HEIGHT}px;
+  ${({ showLimitLine }) =>
+    showLimitLine
+      ? `
   margin: 10px 0px;
-  ${({ theme }) => theme.mediaWidth.upToSmall`
-    margin: 10px 0px 0px 0px;
-  `}
-  .desc {
-    line-height: ${LINE_HEIGHT}px;
-    ${({ showLimitLine }) =>
-      showLimitLine
-        ? `
-    text-overflow:ellipsis;
-    overflow:hidden;
-    display: -webkit-box !important;
-    height: ${LINE_HEIGHT * NUM_LINE_DESC}px;
-    -webkit-line-clamp: ${NUM_LINE_DESC};
-    -webkit-box-orient: vertical;
-    white-space: normal;
-  `
-        : ''}
-  }
+  overflow: hidden;
+  height: ${HEIGHT}px;
+`
+      : `
+  margin: 10px 0px 0px 0px;
+  height: unset;`}
```

Comment:
please format indent for these code, it looks very ugly

Reply:
any extension for it?  I try to align like this 
![image](https://user-images.githubusercontent.com/33005392/176871898-a641ad50-d698-4779-b85f-f8507d3b6ad9.png)

Reply:
wrap with css`` 
maybe it can be auto align for you 
I'm not using vs code but I guess they have styled component extension

Reply:
you can use vscode-styled-components extension for format and IntelliSense

Reply:
thanks guys